docs/docs-content/clusters/edge/trusted-boot/deployment-day2/upgrade-cluster.md

```diff
@@ -36,35 +37,101 @@ Kubernetes layer of the cluster.
   - 32 GB RAM
   - 100 GB storage
 
-:::warning
+- [Git](https://cli.github.com/manual/installation). You can ensure git installation by issuing the git --version
+  command.
 
-Ensure that the keys in the **secure-boot** folder contains all the keys that were used to produce the ISO, including
-the keys in each subdirectory. If you are missing any of the keys, your Edge host will either not be able to boot, or
-not be able to access any data in the persistent partitions.
+## Instructions
 
-:::
+1. If you have access to the original **CanvOS** folder where they provider images were made, and all the necessary keys
+   are available to you in their original folders, skip to step 5.
 
-## Instructions
+2. Check out the [CanvOS](https://github.com/spectrocloud/CanvOS) GitHub repository containing the starter code.
+
+   ```bash
+   git clone https://github.com/spectrocloud/CanvOS.git
+   ```
+
+3. Change to the **CanvOS/** directory.
+
+   ```bash
+   cd CanvOS
+   ```
+
+4. View the available [git tag](https://github.com/spectrocloud/CanvOS/tags).
+
+   ```bash
+   git tag
+   ```
+
+5. Check out the newest available tag. This guide uses the tag **v4.4.0** as an example.
+
+   ```shell
+   git checkout v4.4.0
+   ```
+
+6. Move all keys into folders where they are expected. Ensure all keys that were used to build the ISO are present in
+   the **secure-boot** folder. The following files and folder structure are expected with the following file
+   permissions.
+
+   ```
```

Comment:
Let's seperate the command and the output. That way we can use hideClipboard on the output :)